### DISCUSSION TITLE
<Your ques How-to-Downgrade-apps-on-AppStore-with-iTunes-and-Charles-Proxytion or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by keitht1958** [community/community#124748] [Programming Help]
### Select Topic Area

Question

### Body

[How-to-Downgrade-apps-on-AppStore-with-iTunes-and-Charles-Proxy](https://github.com/qnblackcat/How-to-Downgrade-apps-on-AppStore-with-iTunes-and-Charles-Proxy)
followed all the steps but tried 4 versions of sonos app iTunes says downloading but never completes then Charles timesout
must be doing something wrong

**Comment by LiteBrite82** [comment]
Hi @keitht1958 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project.

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

I'm going to send this over to "programming help" as it may be a better place for your question. 

Best of luck!

**Comment by ebndev** [comment]
Hi @keitht1958,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project,  such as opening an issue in the project's repository or seeing if they have a support page. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. 

We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
